### PR TITLE
Story/vgnwb 219

### PR DESCRIPTION
**Pull request by tjquinn1**
Moved concept types to settings to allow them to be configurable when necessary.

### CODE REVIEW

File: vogon/settings.py [hunk 1]
```diff
@@ -245,3 +245,15 @@
 LOGLEVEL = os.environ.get('LOGLEVEL', 'DEBUG')
 
 SESSION_COOKIE_NAME = 'vogon'
+
+
+# Concept types
+PERSONAL = os.environ.get('PERSONAL', '986a7cc9-c0c1-4720-b344-853f08c136ab')
+CORPORATE = os.environ.get('CORPORATE', '3fc436d0-26e7-472c-94de-0b712b66b3f3')
+GEOGRAPHIC = os.environ.get('GEOGRAPHIC', 'dfc95f97-f128-42ae-b54c-ee40333eae8c')
```
Comment (jdamerow) [inline vogon/settings.py]: the environment variables should be called something that makes it clear what they are for. 'personal' could be anything.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by tjquinn1** [comment]
Made variable names more verbose

File: .inputrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+set completion-ignore-case on
+set show-all-if-ambiguous on
+TAB: menu-complete
```
Comment (jdamerow) [inline .inputrc]: remove this file

File: vogon/settings.py [hunk 3]
```diff
@@ -245,3 +245,15 @@
 LOGLEVEL = os.environ.get('LOGLEVEL', 'DEBUG')
 
 SESSION_COOKIE_NAME = 'vogon'
+
+
+# Concept types
+PERSONAL_CONCEPT_TYPE = os.environ.get('PERSONAL', '986a7cc9-c0c1-4720-b344-853f08c136ab')
+CORPORATE_CONCEPT_TYPE = os.environ.get('CORPORATE', '3fc436d0-26e7-472c-94de-0b712b66b3f3')
+GEOGRAPHIC_CONCEPT_TYPE = os.environ.get('GEOGRAPHIC', 'dfc95f97-f128-42ae-b54c-ee40333eae8c')
```
Comment (jdamerow) [inline vogon/settings.py]: the environment variables are still called "PERSONAL", etc.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by tjquinn1** [comment]
Removed .inputrc file and added more verbose environment names

**Review by jdamerow — APPROVED** [review]

**Pull request merged**